Added IKS02A1_Audio_Button example (#13)
stm32duino/FP_Examples: README.md

provided by the X-NUCLEO-IKS01A3 between two NUCLEO boards connected each other through S2-LP based Expansion Boards; in order to send the sensor data 
you just need to push the button of the NUCLEO board; in order to see the received sensor data you just need to open a hyperterminal connected to 
the Virtual COM port of the Nucleo board; the LED of the NUCLEO board will blink after every successful transmission or reception.

> * X_NUCLEO_IKS02A1_Audio_Button: This application provides a simple example of recording audio from a X-NUCLEO-IKS02A1, convert from PDM to PCM, then save the audio like a file wave in an SD Card. 

Reviewer: ```suggestion
* X_NUCLEO_IKS02A1_Audio_Button: This application provides a simple example of recording audio from a X-NUCLEO-IKS02A1, converting from PDM to PCM, then saving it as a wave file to an SD Card. 
```
